services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/index.js

```diff
@@ -67,6 +67,22 @@ const Library = ({ libraryData, params }) => {
   const { sponsor } = libraryData.params
   const SponsorIcon = teams[sponsor] ? teams[sponsor].icon : Events
 
+  let demoLinks = '–'
+  if (libraryData.content.demoLinks) {
```

Comment:
What about this logic being in a DemoLinks component (`/components/demo-links.js`) that takes `links` as a prop and renders a list of links with fallback en dash? So you could use it for both library and asset page?

https://github.com/carbon-design-system/carbon-platform/blob/main/docs/resource-schemas.md#shared-schemas

It's a shared schema between library and asset.